packages/sui-ssr/bin/sui-ssr-build.js

```diff
@@ -70,10 +70,8 @@ const build = () =>
         jsonStats.warnings.map(warning => console.log(warning))
       }
 
-      const SERVER_ENTRY_POINT = path.join(
-        BUILD_SERVER_PATH,
-        jsonStats.assetsByChunkName.main
```

Comment:
webpack@4, it was a string: https://v4.webpack.js.org/api/stats/
webpack@5, it is an array: https://webpack.js.org/api/stats/